CMakeLists.txt

```diff
@@ -37,12 +37,11 @@ option(BUILD_TESTS          "Build the unit tests." OFF)
 option(BUILD_TCMALLOC       "Build with tcmalloc" OFF)
 option(BUILD_PROFILER       "Enable google cpu and heap profiler support" OFF)
 option(BUILD_SYMBOLS        "Build with Symbols" OFF)
-option(BUILD_EFENCE         "Build with efence" OFF)
 option(BUILD_ASAN           "Build with Address Sanitizer" OFF)
 option(BUILD_UBSAN          "Build with Undefined Behavior Sanitizer" OFF)
 option(BUILD_UDNS           "Build with udns (async dns support)" ON)
-option(BUILD_UBUNTU         "Build for Ubuntu" OFF)
-option(BUILD_CUSTOM_OPENSSL "Build for Ubuntu" OFF)
+option(BUILD_LINUX         "Build for Ubuntu" OFF)
```

Comment:
Suggest:
`Build for Linux`

instead of 
`Build for Ubuntu`

Reply:
Nice catch -fixin